### PR TITLE
I1696 che ifeval application

### PR DESCRIPTION
**Pull request by MichalMaler**
Signed-off-by: Michal Maléř mmaler@redhat.com

A version of already accepted PR in the master branch.

Fixing a broken link.
Specifying the upstream content.
Adding information about mother assembly.
Adding the :page-liquid: attribute to assembly metadata

### CODE REVIEW

File: src/main/pages/che-7/administration-guide/con_database-setup.adoc [hunk 1]
```diff
@@ -14,7 +14,7 @@ summary:
 [id="database-setup_{context}"]
 = Database setup
 
-You can use PostgreSQL database for persisting data about the state of {prod-short}. It contains information about the user accounts, workspaces, preferences, ssh keys, and so on.
+PostgreSQL database is used for persisting data about the state of {prod-short}. It contains information about the user accounts, workspaces, preferences, ssh keys, and so on.
```
Comment (yhontyk) [inline src/main/pages/che-7/administration-guide/con_database-setup.adoc:17]: "is used" by what or who?

Specify, e.g. Eclipse Che uses PostreSQL....

File: src/main/pages/che-7/end-user-guide/con_workspace-exposure-strategies.adoc [hunk 2]
```diff
@@ -35,4 +35,4 @@ Therefore, only applications that use relative URLs in their UI work with the si
 
 == Default-host strategy
 
-This strategy exposes the components to the outside world on the subpaths of the default host of the cluster. It is similar to the single-host strategy. All the limitations and advantages of the single-host strategy applying to this strategy as well.
+This strategy exposes the components to the outside world on the sub-paths of the default host of the cluster. It is similar to the single-host strategy. All the limitations and advantages of the single-host strategy applying to this strategy as well.
```
Comment (yhontyk) [inline src/main/pages/che-7/end-user-guide/con_workspace-exposure-strategies.adoc:38]: Do not hyphen.

File: src/main/pages/che-7/end-user-guide/con_workspace-exposure-strategies.adoc [hunk 3]
```diff
@@ -23,7 +23,7 @@ On a {prod-short} server secured using the Transport Layer Security (TLS) protoc
 
 == Single-host strategy
 
-This strategy is available on Kubernetes, but not on OpenShift. When it is used, all workspaces are deployed to subpaths of the main {prod-short} server domain.
+This strategy is available on Kubernetes, but not on OpenShift. When it is used, all workspaces are deployed to sub-paths of the main {prod-short} server domain.
```
Comment (yhontyk) [inline src/main/pages/che-7/end-user-guide/con_workspace-exposure-strategies.adoc:26]: Do not hyphen.

File: src/main/pages/che-7/extensions/assembly_eclipse-che4z.adoc [hunk 4]
```diff
@@ -21,7 +21,7 @@ Developers can now find the code they need to work on in Explorer for Endevor, G
 
 Eclipse Che4z is an open source project. Many of the components of the Che4z stack are also available on the VS Code marketplace as part of the https://marketplace.visualstudio.com/items?itemName=broadcomMFD.code4z-extension-pack[Code4z] package.
 
-== Use Cases
+== Use cases
```
Comment (yhontyk) [inline src/main/pages/che-7/extensions/assembly_eclipse-che4z.adoc:24]: I'm pretty sure I fixed this one... Don't forget to rebase from time to time so we don't do double work. :D

File: src/main/pages/che-7/overview/con_persistent-volume-configuration.adoc [hunk 5]
```diff
@@ -28,8 +28,8 @@ User-defined PVCs are created with few modifications:
 
 - they are provisioned with generated names to guarantee that it is not conflicting with other PVCs in namespace;
 
-- subpaths of mount volumes that reference user-defined PVCs are prefixed with `{workspace id}/{PVC name}`.
-It is done to have the same data structure on PV on different PVC strategies(link:#how-the-subpaths-are-used[more]);
+- sub-paths of mount volumes that reference user-defined PVCs are prefixed with `{workspace id}/{PVC name}`.
```
Comment (yhontyk) [inline src/main/pages/che-7/overview/con_persistent-volume-configuration.adoc:31]: Do not hyphen. The rule of thumb is to not hyphen whenever you can (if it doesn't affect the meaning of the word or doesn't create ambiguity).

**Review by yhontyk — CHANGES_REQUESTED** [review]

**Review by yhontyk — APPROVED** [review]

**Comment by MichalMaler** [comment]
This has to be canceled. Creating a new PR

**Comment by yhontyk** [comment]
Closing in favour of https://github.com/eclipse/che-docs/pull/1141

**Pull request closed without merging**